crontab/notify_old_pp.php

```diff
@@ -10,93 +10,132 @@
+        {
+            $projects[$checkedoutby] = [];
+        }
+        $projects[$checkedoutby][] = $row;
+    }
+
+    return $projects;
+}
+
+function send_pp_reminders($checkedoutby, $projects)
+{
+    global $code_url, $general_help_email_addr, $forums_url,
+           $site_name, $site_signoff, $site_abbreviation;
+
+    $urlbase = "$code_url/project.php?id=";
+
+    $numprojs = count($projects);
+
+    echo "\nNotifying $checkedoutby about $numprojs project(s):\n";
+
+    $projects_list = "";
+    foreach($projects as $project)
+    {
+        $nameofwork = $project["nameofwork"];
+        $authorsname = $project["authorsname"];
+        $projectid = $project["projectid"];
+        $modifieddate = strftime("%e %B %Y", $project["modifieddate"]);
+        $lastvisitdate = strftime("%e %B %Y", $project["lastvisitdate"]);
+
+        $url = $urlbase . $projectid;
+        $work_details  = "$nameofwork by $authorsname ($projectid)";
+        $time_details = "[checked out since $modifieddate, project last visited $lastvisitdate]";
+        $projects_list .= "$work_details\n$time_details\n    $url\n\n";
+
+        echo "    $work_details\n";
     }
 
-    $numprojs++;
+    $user = new User($checkedoutby);
+    $email = $user->email;
 
-    $url = $urlbase . $projectid;
+    // only one project
+    if($numprojs == 1)
+    {
+        $url = $urlbase . $projects[0]["projectid"];
 
-    $projectslist .= "$nameofwork by $authorsname ($projectid), out since $nicedate\n$url\n\n";
-    if ($numprojs == 1) {
         $subject = "$site_abbreviation: Status update needed for 1 project checked out for PPing over 90 days";
-    } else {
+
+        $message = <<<EOF
+Hello $checkedoutby,
+
+Our database indicates that you have had a PP project checked out for more than 90 days:
+
+$projects_list
+If you haven't yet finished and wish to continue working on this book, please visit $url. This will update the status of the project. If you need help please forward a copy of this email (quoting the information on the book, above) with a brief description of the status to $general_help_email_addr.
+
+If you have completed your work on the book, please visit $url. Select the 'Upload for verification' option and follow the prompts. You will be able to leave a message for the verifier during this process, if you have any special information or comments to pass on.
+
+If you are waiting on missing images or page scans, please add the details to the Missing Page Wiki at: $forums_url/viewtopic.php?t=7584
+
+If you no longer wish to have this text assigned to you please visit the $site_name website Post Processing section and select Return to Available for this book, or forward this email to $general_help_email_addr and state that you would no longer like to have the book in question assigned to you so that we may return it to the available pool for someone else to work on.
+
+$site_signoff
+EOF;
+    }
+    // more than one project
+    else
```

Comment:
Wouldn't `else // more than one project` read better?

Personally, I'd probably put the comment in the block:
```
else
{
    // more than one project
    ....
```

Looks like [our style quide](https://www.pgdp.net/wiki/DP_Code_Best_Practices#Comments) doesn't address this.